spynnaker/pyNN/models/neuron/neuron_models/neuron_model_external_device_control.py

```diff
@@ -0,0 +1,230 @@
+# Copyright (c) 2017-2019 The University of Manchester
```

Comment:
I'm confused about the need for this entire file.  It looks like a clone of LIF.  Why?

Reply:
was when thinking a different route. killing